module/zfs/metaslab.c

```diff
@@ -1672,7 +1672,7 @@ metaslab_df_alloc(metaslab_t *msp, uint64_t size)
 	uint64_t align = size & -size;
 	uint64_t *cursor = &msp->ms_lbas[highbit64(align) - 1];
 	range_tree_t *rt = msp->ms_allocatable;
-	int free_pct = range_tree_space(rt) * 100 / msp->ms_size;
+	uint32_t free_pct = range_tree_space(rt) * 100 / msp->ms_size;
```

Comment:
Why is it uint32_t if metaslab_df_free_pct is uint_t?

Reply:
It looked consistent with the other stack variables, but I will change it.